### PR TITLE
about-page and navigation

### PR DESCRIPTION
**Pull request by pung-one**
- add about page
- change navigation to make it more clear
- moved handleUpdateFavs from toggle function to useEffect hook

### CODE REVIEW

File: components/ColorPicker/index.js [hunk 1]
```diff
@@ -12,11 +13,18 @@ const StyledForm = styled.form`
   gap: 3vh;
 `;
 
+const Arrow = styled(ImArrowDown)`
```
Comment (bk1405bgl) [inline components/ColorPicker/index.js:16]: i think, you dont't need this anymore
ArrowDown

also in Line 6 the import is not needed anymore?

**Review by bk1405bgl — CHANGES_REQUESTED** [review]
Please chekc comments

File: components/ColorPicker/index.js (hunk 1 above)
Comment (pung-one) [reply components/ColorPicker/index.js:16]: thank you, but i do need that arrow. it's another one than the one we talked about :)

**Review by bk1405bgl — APPROVED** [review]
looks very good

**Pull request merged**